#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_debian-patch.dpatch by Christian Marillat <marillat@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: libdts patch for libavcodec.

@DPATCH@

--- mythtv-0.18.1.orig/libs/libavcodec/libavcodec.pro.orig 2006-02-15 05:36:11.000000000 -0800
+++ mythtv-0.18.1/libs/libavcodec/libavcodec.pro      2006-02-15 05:36:23.000000000 -0800
@@ -317,7 +317,7 @@
 
 contains( CONFIG_DTS, yes ) {
     SOURCES += dtsdec.c
-    LIBS += -ldts
+    LIBS += -ldts_pic
 }
 
 using_xvmc {
